.travis.yml

```diff
@@ -1,11 +1,27 @@
 language: ruby
 sudo: false
+env:
+  global:
+    - secure: "gxjwgl9l9ZP/1t86odZyMzWyiVBkG1oFzeL+Kb6z9Dtfu954l9RbytlenTTcnRw1KtaDBZVBs0ix3K45b8T2S6JFzLQ88Z/36DmXGIN6vxctfnVD9sqbbyOuP++RhibK+qUqbPRHo6a3wBsMQ87DMj5S6pJe3dxbCKRXa8zcMd0="
+before_install:
+  - openssl aes-256-cbc -K $encrypted_df4ab1bff570_key -iv $encrypted_df4ab1bff570_iv -in .travis/secrets.tar.enc -out .travis/secrets.tar -d
```

Comment:
This will need to move the deploy steps from `before_install` to `before_deploy`, as ENV vars won't be available when running CI for forked PRs, which are common on this repo.

See https://github.com/alphagov/govuk_frontend_toolkit/pull/385